app/views/logs/_log_list.html.erb

```diff
@@ -1,7 +1,8 @@
 <h2 class="govuk-body">
   <%= render(SearchResultCaptionComponent.new(searched:, count: pagy.count, item_label:, total_count:, item: "logs", path: request.path)) %>
   <% if logs&.first&.lettings? %>
-    <%= govuk_link_to "Download (CSV)", csv_download_url, type: "text/csv" %>
+    <%= govuk_link_to "Download (CSV)", csv_download_url, type: "text/csv", class: "govuk-!-margin-right-4" %>
+    <%= govuk_link_to "Download (CSV, codes only)", csv_codes_only_download_url, type: "text/csv" %>
```

Comment:
would this display for all users (providers and coordinators etc)?

Reply:
The ticket doesn't mention anything about this so I assumed that this type of download would be available with the same permissions as the old download

Reply:
this is now done